faucet/urls.py

```diff
@@ -64,4 +65,5 @@
         name="chain-balance",
     ),
     path("user/donation", DonationReceiptView.as_view(), name="donation-receipt"),
+    path("leader-board/gas-tap", LeaderBoardView.as_view(), name="token-tap-leader-board")
```

Comment:
please make this
"gas-tap/leaderboard/"

and also add the trailing slash for the user/donation -> user/donation/